cdap-ui/app/cdap/components/PreviewData/TableContainer.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import If from 'components/If';
+import { ITableElements } from 'components/PreviewData';
+import { INode } from 'components/PreviewData/utilities';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import classnames from 'classnames';
+import T from 'i18n-react';
+
+const I18N_PREFIX = 'features.PreviewData.TableContainer';
+
+const styles = (theme) => ({
+  outerContainer: {
+    display: 'flex',
+  },
+  innerContainer: {
+    overflow: 'scroll',
+    padding: '10px',
+    width: '100%',
+  },
+  split: {
+    maxWidth: '50%',
+    borderBottom: `1px solid ${theme.palette.grey[400]}`,
+    padding: '10px',
+    borderRight: `1px solid ${theme.palette.grey[400]}`,
+    '& :last-of-type': {
+      borderRight: 0,
+    },
+  },
+  h2Title: {
+    fontSize: '1.4rem !important',
+    fontWeight: 'bold' as 'bold',
+    marginBottom: 0,
+    borderBottom: `1px solid ${theme.palette.grey[400]}`,
+    paddingBottom: '5px',
+    paddingLeft: '10px',
+    marginLeft: '-10px',
+    marginRight: '-10px',
+    marginTop: '0',
+  },
+});
+
+interface IPreviewTableContainerProps extends WithStyles<typeof styles> {
+  dataTables: ITableElements;
+  selectedNode: INode;
+}
+
+const TableContainer: React.FC<IPreviewTableContainerProps> = ({
+  classes,
+  dataTables,
+  selectedNode,
+}) => {
+  return (
+    <div className={classes.outerContainer}>
+      <If condition={!selectedNode.isSource}>
+        <div
+          className={classnames(classes.innerContainer, {
+            [classes.split]: !selectedNode.isSource && !selectedNode.isSink,
+          })}
+        >
+          <h2 className={classes.h2Title}>{T.translate(`${I18N_PREFIX}.inputHeader`)}</h2>
+          {dataTables.input}
+        </div>
+      </If>
+      <If condition={!selectedNode.isSink}>
+        <div
+          className={classnames(classes.innerContainer, {
+            [classes.split]: !selectedNode.isSource && !selectedNode.isSink,
+          })}
+        >
+          <h2 className={classes.h2Title}>{T.translate(`${I18N_PREFIX}.outputHeader`)}</h2>
+          {dataTables.output}
+        </div>
+      </If>
+    </div>
+  );
+};
+
+const StyledTableContainer = withStyles(styles)(TableContainer);
+
+function PreviewTableContainer(props) {
```

Comment:
is this function necessary? I think you can just simply export the StyledTableContainer

Reply:
Done!